internal/repos/store.go

```diff
@@ -742,6 +744,21 @@ func (s *store) ListSyncJobs(ctx context.Context, opt SyncJobsListOptions) ([]Sy
 	return scanJobs(rows)
 }
 
+func (s *store) SetSyncJobState(ctx context.Context, state string, ids ...int64) (err error) {
```

Comment:
Named error is unused

```suggestion
func (s *store) SetSyncJobState(ctx context.Context, state string, ids ...int64) error {
```